modules/workflow-workflowoperation/src/main/java/org/opencastproject/workflow/handler/workflow/AnalyzeTracksWorkflowOperationHandler.java

```diff
@@ -70,7 +71,8 @@ public WorkflowOperationResult start(WorkflowInstance workflowInstance, JobConte
     final String sourceFlavor = getConfig(workflowInstance, OPT_SOURCE_FLAVOR);
```

Comment:
Why also have the configuration in here. Your update is supposed to remove these I think.

Reply:
True, I must have missed these. Removed the unnecessary/old variables.